### ISSUE TITLE
A11y statement: list WCAG criterias we aren't testing or complying with

### ISSUE THREAD

**Issue by mindrones** [nestauk/eurito_indicators#138]
We need to be explicit about WCAG criterias we aren't testing or complying with for lack of time/resources. This is especially important for features that require manual testing.

See #27
